frontend/src/components/Gamemodes/Gamemodes.module.scss

```diff
@@ -0,0 +1,18 @@
+@import "../../variables";
+
+.mainContent {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 50px;
+  flex-shrink: initial;
+  margin: 0 75px;
+}
+
+.title {
```

Comment:
for stuff like this I think we should probably have a global class so that all titles (or titles on sheets at least) have the same style